bin/update

```diff
@@ -28,18 +40,20 @@ chdir APP_ROOT do
   # If we need a "reset all the things" script, we should call it bin/reset.
   system! 'bundle update'
 
+  # load 'Rakefile'
+  load_rakefile
   puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
+  Rake::Task["db:migrate"].invoke
 
   puts "\n== Seeding database =="
-  system! "bin/rails db:seed GOOD_MIGRATIONS=skip"
+  Rake::Task["db:seed"].invoke
 
   puts "\n== Resetting tests =="
   system! "bin/rails test:vmdb:setup"
 
   unless ENV["SKIP_AUTOMATE_RESET"]
     puts "\n== Resetting Automate Domains =="
-    system! "bin/rails evm:automate:reset"
+    Rake::Task["evm:automate:reset"].invoke
```

Comment:
@NickLaMuro To be frank, this is most of the problem.  This rake task should detect if it needs to do anything and only reset if it's needed.  I'd rather not make all developers suffer through this by hiding it via the ENV variable locally.